ParisBNF/abb/BNFabb119.xml

```diff
@@ -320,7 +320,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                       <title type="complete" ref="LIT3528Epistle#Introduction">Epistle to Titus</title>
                                     <textLang mainLang="gez"/>
                                 </msItem>
-                                </msItem>
```

Comment:
still nesting issue - remove one /msItem before 3.6 (line 311) and insert one /msItem before 3.7

Reply:
done